libs/asset-viewer/src/lib/services/viewer-params.service.ts

```diff
@@ -20,13 +21,16 @@ export class ViewerParamsService {
   private readonly store = inject(Store<AppStateWithAssetSearch>);
 
   async readParamsFromStore(): Promise<ViewerParams> {
-    const state = await firstValueFrom(
+    const searchState = await firstValueFrom(
       this.store.pipe(map((store) => store.assetSearch)) as Observable<AssetSearchState>
     );
+    const sharedState = await firstValueFrom(
+      this.store.pipe(map((store) => store.shared)) as Observable<AppSharedState>
+    );
```

Comment:
Probably a good idea to unify these.
```suggestion
    const [searchState, sharedState] = await firstValueFrom(
      this.store.pipe(map((store) => [store.assetSearch, store.shared] as [AssetSearchState, AppSharedState]))
    );
```